src/runtime/resource/detail/detail_partitioner.cpp

```diff
@@ -361,7 +361,7 @@ namespace hpx { namespace resource { namespace detail
                         // exclusively if dynamic pools are enabled.
                         // Also, by default, the first PU is always exclusive
                         // (to avoid deadlocks).
-                        add_resource(p, "default",
+                        add_resource(p, initial_thread_pools_[0].pool_name_,
```

Comment:
May I suggest you use the `get_default_pool_name()` member function everywhere here instead of the `initial_thread_pools_[0].pool_name_` you currently use?

Reply:
inside detail::partitioner I used initial_thread_pools_[0].pool_name, but outside it, I used get_default_pool_name. Not a big deal. I will change it.